### ISSUE TITLE
Anyone can access web monitoring website?

### ISSUE THREAD

**Issue by tmann50** [diladele/webproxy#236]
It looks like anyone on the machine that the proxy is running on can access the monitoring portal, regardless of their user status.

Its accessible at 127.0.0.1:8000 or localhost:8000 on the machine by any user. Would it be possible to add authentication to this site?

**Comment by ra-at-diladele-com** [comment]
Yes it was supposed that for Windows Server machines only admins are able to login, thus if you are an admin there is nothing wrong is showing you the traffic monitor. But I will move this onto the next version list - maybe it is possible to use Negotiate auth there too.
